test

```diff
@@ -543,8 +544,27 @@ function staticcheck_pass {
 }
 
 function revive_pass {
-	if command -v revive >/dev/null; then
-		reviveResult=$(revive -config ./tests/revive.toml -exclude "vendor/..." ./... 2>&1 || true)
+	REVIVE=revive
+
+	if ! command -v $REVIVE >/dev/null; then
+		echo "Installing revive $REVIVE_VERSION"
+		if [ "$GOARCH" == "amd64" ]; then
+			URL="https://github.com/mgechev/revive/releases/download/${REVIVE_VERSION}/revive_linux_amd64.tar.gz"
+		elif [[ "$GOARCH" == "arm" || "$GOARCH" == "arm64" ]]; then
+			URL="https://github.com/mgechev/revive/releases/download/${REVIVE_VERSION}/revive_linux_arm64.tar.gz"
+		else
+			echo "Unsupported architecture: $GOARCH"
+			exit 255
+		fi
```

Comment:
If the local OS is macOS, then contributors need to manually install the tool beforehand.